src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerAttribute.cs

```diff
@@ -144,8 +144,14 @@ public CosmosDBTriggerAttribute(string databaseName, string collectionName)
         /// Values should be comma-separated.
         /// </summary>
         /// <example>
-        /// PreferredLocations = "East US,South Central US,North Europe"
+        /// PreferredLocations = "East US,South Central US,North Europe".
         /// </example>
         public string PreferredLocations { get; set; }
+
+        /// <summary>
+        /// Optional.
+        /// Enable to use with Multi Master accounts.
```

Comment:
Maybe add something that this only applies to the lease collection?

Reply:
Done